integrations/clickup/main.py

```diff
@@ -0,0 +1,53 @@
+from enum import StrEnum
+from clickup.client import ClickupClient
+from loguru import logger
+from port_ocean.context.ocean import ocean
+from port_ocean.core.ocean_types import ASYNC_GENERATOR_RESYNC_TYPE
+
+
+class ObjectKind(StrEnum):
+    TEAM = "team"
+    PROJECT = "project"
+    ISSUE = "issue"
+
+
+@ocean.on_resync(ObjectKind.TEAM)
+async def on_resync_teams(kind: str) -> ASYNC_GENERATOR_RESYNC_TYPE:
+    logic_settings = ocean.integration_config
+    client = ClickupClient(
+        logic_settings["clickup_url"],
+        logic_settings["clickup_apikey"]
+    )
```

Comment:
You are referencing this several times. You can put it in a function and call it